CONTRIBUTING.md

```diff
@@ -182,15 +182,8 @@ Cluster API maintains the most recent release/releases for all supported API and
 
 ### Removal of v1alpha3 & v1alpha4 apiVersions
 
-Both v1alpha3 and v1alpha4 have been removed from Cluster API as of release 1.7.
-
-For more details and latest information please see the following issue: [Removing v1alpha3 & v1alpha4 apiVersions](https://github.com/kubernetes-sigs/cluster-api/issues/8038).
-
-Note: Removal of a deprecated APIVersion in Kubernetes [can cause issues with garbage collection by the kube-controller-manager](https://github.com/kubernetes/kubernetes/issues/102641)
```

Comment:
I think this note is still true

Reply:
No issue to keep it around, but given that we are not removing API, it is less relevant than before.

Reply:
I think this note is already a problem because of the unserving. That we don't remove the versions doesn't change the impact (IIRC)

Reply:
(the garbage collector won't be able to get the old apiVerisons if they are not served anymore)